packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html

```diff
@@ -15,13 +15,13 @@
         </oui-datagrid-column>
         <oui-datagrid-column
             data-title="'pci_projects_project_storages_cold_archive_containers_creation_date_label' | translate"
-            data-property="creationDate"
-            data-type="string"
+            data-property="gridPropertyCreationDate"
+            data-type="date"
             data-searchable
             data-sortable
             data-filterable
         >
-            <span data-ng-bind="::$row.creationDate"></span>
+            <span data-ng-bind="$row.gridDisplayCreationDate"></span>
```

Comment:
**nitpick:** Use Angular.js filter to display date instead adding a new key to the object.

Reply:
Done.